build.ps1

```diff
@@ -45,7 +48,8 @@ param (
    [Switch] $List,
    [String] $Name,
    [String] $Tag,
-   [String] $Registry
+   [String] $Registry,
```

Comment:
Documentation a few lines above:
```
    .PARAMETER Registry
    Set the container registry. Defaults to dockerhub under steeltoeoss.
```
Should this be updated to something other than dockerhub?

Reply:
While publishing to dockerhub has been removed, this parameter documentation is still accurate